ocpmodels/common/relaxation/ase_utils.py

```diff
@@ -30,12 +30,12 @@ def batch_to_atoms(batch):
     natoms = batch.natoms.tolist()
     numbers = torch.split(batch.atomic_numbers, natoms)
     fixed = torch.split(batch.fixed, natoms)
-    forces = torch.split(batch.force, natoms)
+    forces = torch.split(batch.force, natoms) if "force" in batch else None
     positions = torch.split(batch.pos, natoms)
     tags = torch.split(batch.tags, natoms)
     cells = batch.cell
-    energies = batch.y.tolist()
-
+    energies = batch.y.tolist() if "y" in batch else batch.y_init.tolist()
```

Comment:
IS2RE also requires `y_relaxed`. Maybe we can feed in an argument to control the task here rather than the else statements.